contrib/win32/win32compat/misc.c

```diff
@@ -191,7 +191,7 @@ nanosleep(const struct timespec *req, struct timespec *rem)
  * Copyright (c) 2009, 2010 NoMachine
  * All rights reserved
  */
-static int
```

Comment:
why are these not static anymore?

Reply:
Also related to the symbols no longer being exported by LibreSSL. Updating to match internal declarations:
https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/win32compat/inc/strings.h
https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/win32compat/inc/sys/time.h